views/AUGSD/admin/portals/feedbacks/index.jade

```diff
@@ -65,4 +65,15 @@ block content
 							.mdl-cell.mdl-cell--4-col.mdl-card.mdl-shadow--2dp.profile-select(onclick="window.location.replace('#{rootURL}/#{reqPortal}/midsem')")
 								img.profile(src='/images/feedback/sem.png')
 								h4.center.custom Mid-Sem Feedback
+							.mdl-layout-spacer
+					.mdl-typography--font-light.mdl-typography--display-1-color-contrast.center Export Feedbacks
+					p.mdl-typography--font-light
+						.mdl-grid
+							.mdl-layout-spacer
+							.mdl-cell.mdl-cell--4-col.mdl-card.mdl-shadow--2dp.profile-select(onclick="window.location.replace('#{rootURL}/#{reqPortal}/export/24x7')")
+								img.profile(src='/images/feedback/24.png')
```

Comment:
Use a different image for the export link. Divide the view into two halves. 

Upper half for:

1. View 24x7 Feedbacks (Use existing image)
1. Export 24x7 Feedbacks (Use a new import/export image)

Lower Half For

1. View Mid-Semester Feedbacks (Use existing image)
1. Export Mid-Semester Feedbacks (Use the new import/export image)